FIDO2_messages.md

```diff
@@ -0,0 +1,21 @@
+## [](#header-1) Communications Between Aux and Main MCU
+This describes the FIDO2 messages sent between Aux MCU and Main MCU.
+
+## [](#header-2) Message Structure and Serial Link Specs 
+Each FIDO2 message is wrapped in the header defined as follows"
+
+| byte 0 - 1         | byte 2 - 3        | byte 4 - 551  |
+|:-------------------|:------------------|:--------------|
+| FIDO2 Message Type | Reserved          | Payload       |
+
+**FIDO2 Message Type:**
+- 0x0001: [FIDO2 Authenticate Credential Request](FIDO2_messages_details.md#fido2-authenticate-credential-request)
+- 0x0002: [FIDO2 Authenticate Credential Response](FIDO2_messages_details.md#fido2-authenticate-credential-response)
+- 0x0003: [FIDO2 Make Authenticator Data Request](FIDO2_messages_details.md#fido2-make-authenticator-data-request)
+- 0x0004: [FIDO2 Make Authenticator Data Response](FIDO2_messages_details.md#fido2-make-authenticator-data-response)
+- 0x0005: [FIDO2 Get Next Credential Request](FIDO2_messages_details.md#fido2-get-next-credential-request)
```

Comment:
from our previous conversation, I think this is not needed anymore

Reply:
Agree.